src/main/java/com/simibubi/create/content/kinetics/fan/processing/FanProcessingType.java

```diff
@@ -16,7 +18,7 @@
 import net.minecraft.world.phys.Vec3;
 
 public interface FanProcessingType {
-	boolean isValidAt(Level level, BlockPos pos);
+	boolean isValidAt(LevelAccessor level, BlockPos pos);
```

Comment:
Please don't change the parameter type here since it's going to be an unnecessary breaking change on a widely used API.

Instead, make checkcasts in `PeculiarBellBlock#updateShape`, we don't need it to convert during worldgen so it should be safe to do so.